docs/operation-guides/config-09-apigateway-security_options.md

```diff
@@ -1,22 +1,26 @@
 ---
-title: Available security options
-type: Details
+title: Configure Authorization (OAuth2, JWT)
+type: Configuration
```

Comment:
I'm wondering whether it should be security that KK mentioned today 🤔

Reply:
I think so!